docs/organizer-guide/confs/community-events.rst

```diff
@@ -0,0 +1,20 @@
+Community Events
+================
+
+We are happy to promote events that are created by our community.
+They are **not** officially organized by Write the Docs organizers,
+but they are sister events that share a similar goal.
+
+Requirements
+------------
+
+In order to be listed as a community event,
+we require a few things.
+This is primarily to make sure people understand who is organizing the event,
+and to make sure companies can't run events inside our community for their own profit.
+
+Community events:
+
+* Must have a Code of Conduct
+* Must specify they **are not** official Write the Docs events, but must link back to Write the Docs as a refenence to the community
+* Must list the organizers, and they must be from multiple companies
```

Comment:
and they must > who must

Reply:
s/refenence/reference/

Reply:
> as a refenence to the community

what does this even mean? :-p

Reply:
* Must have multiple clearly listed organizers from more than one company  

^ still not very happy with that bulletpoint